bot/cogs/help_channels.py

```diff
@@ -548,6 +552,22 @@ async def move_to_dormant(self, channel: discord.TextChannel, caller: str) -> No
 
         A caller argument is provided for metrics.
         """
+        msg_id = await self.question_messages.pop(channel.id)
+
+        # When message ID exist in cache, try to get it from cache first. When this fail, use API request.
+        # When this return 404, this mean that message is deleted and can't be unpinned.
+        if msg_id:
+            msg = discord.utils.get(self.bot.cached_messages, id=msg_id)
+            if msg is None:
+                try:
+                    msg = await channel.fetch_message(msg_id)
+                except discord.NotFound:
+                    log.debug(f"Can't unpin message {msg_id} because this is deleted.")
+
+            # When we got message, then unpin it
+            if msg:
+                await msg.unpin()
```

Comment:
PR looks solid to me. However the lines 560-569 could be simplified, discord.py has a `HTTPClient.unpin_method` method [seen here](https://github.com/Rapptz/discord.py/blob/master/discord/http.py#L450) which is exposed under `bot.http` which we could use instead, as that will directly be making call to API without needing to try and find the message in either cache or via `fetch_message` API call and will only need to handle `HTTPException` error, which will be just raised when message wasn't found.